k8s/deployment-demo.yml

```diff
@@ -0,0 +1,282 @@
+apiVersion: extensions/v1beta1
```

Comment:
feeling out of my depth with this file 😅  there doesn't need to be a config for prod though?

Reply:
This is a duplicate of the production deployment.yaml file, with values adjusted for demo!